config/200-config.yaml

```diff
@@ -49,3 +49,11 @@ data:
     # across multiple layers of TCP proxies.
     # NOTE THAT THIS IS AN EXPERIMENTAL / ALPHA FEATURE
     enable-proxy-protocol: "false"
+
+    # The server certificates to serve the internal TLS traffic for Kourier Gateway.
+    # It is specified by the secret name, which has the "tls.crt" and "tls.key" data field.
+    # Use an empty value to disable the feature (default).
+    #
+    # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
+    #       for now. Use with caution.
+    kourier-internal-cert-secret: ""
```

Comment:
Suggest calling this `cluster` rather than `internal` -- it's a bit clearer about what it's for.

Reply:
Also, I don't think you need the `kourier-` prefix, since this is in `config-kourier` already.